graph/appender/circuit_breaker.go

```diff
@@ -25,18 +25,20 @@ func applyCircuitBreakers(n *tree.ServiceNode, namespaceName string, istioClient
 	istioDetails, err := istioClient.GetIstioDetails(namespaceName, strings.Split(n.Name, ".")[0])
 	if err == nil {
 		if istioDetails.DestinationPolicies != nil {
+			var applicableDps models.DestinationPolicies
 			dps := make(models.DestinationPolicies, 0)
 			dps.Parse(istioDetails.DestinationPolicies)
 			for _, dp := range dps {
 				if dp.CircuitBreaker != nil {
 					if d, ok := dp.Destination.(map[string]interface{}); ok {
 						if d["labels"].(map[string]interface{})["version"] == n.Version {
 							n.Metadata["isCircuitBreaker"] = "true"
-							break // no need to keep going, we know it has at least one CB policy
+							applicableDps = append(applicableDps, dp)
```

Comment:
This is wrong because not all destination policies are circuit breakers, but this code sets isCircuitBreaker == true while caching the dest policy. Destination policies could be things like load balancers, too. See https://istio.io/docs/reference/config/istio.routing.v1alpha1.html#DestinationPolicy.